<launch>

  <include file="$(find kinect_challenge)/launch/sim_p3dx_localize.launch" />
    
  <node type="Mapper.py"    name = "SetWaypoints"    pkg="kinect_challenge" output="screen">
    <param name="MapPath" type="str" value="$(find kinect_challenge)/maps" />
    
    <!-- Waypoints saved during mapping, file in maps folder -->
    <!-- any existing waypoint file is backed up to a file with time stamp appended to the name -->
    <param name="WaypointFile" type="str" value="waypoints.wp" />
    
    <!-- Button index for joystick button used to place a landmark -->
    <param name="lmButtonIdx"    type="int" value="4" />
    
    <!-- Button index for joystick button used to save current map -->
    <!-- map saved to map folder, file name = map_<timestamp>.[png|yaml] -->
    <!-- map  saved to map folder, file name = map.[png|yaml] -->
    <param name="msButtonIdx"    type="int" value="5" />
  </node>
    
</launch>

